controllers/localvolume/api_updater.go

```diff
@@ -121,27 +121,27 @@ func (s *sdkAPIUpdater) applyClusterRoleBinding(roleBinding *rbacv1.ClusterRoleB
 	return resourceapply.ApplyClusterRoleBinding(s.clientset.RbacV1(), events.NewInMemoryRecorder(componentName), roleBinding)
 }
 
-func (s *sdkAPIUpdater) applyStorageClass(sc *storagev1.StorageClass) (*storagev1.StorageClass, bool, error) {
-	return applyStorageClass(s.clientset.StorageV1(), sc)
+func (s *sdkAPIUpdater) applyStorageClass(ctx context.Context, sc *storagev1.StorageClass) (*storagev1.StorageClass, bool, error) {
+	return applyStorageClass(ctx, s.clientset.StorageV1(), sc)
 }
 
 func (s *sdkAPIUpdater) applyDaemonSet(ds *appsv1.DaemonSet, expectedGeneration int64, forceRollout bool) (*appsv1.DaemonSet, bool, error) {
 	if forceRollout {
 		klog.Infof("Rolling out DaemonSet: %s/%s", ds.Name, ds.Namespace)
 	}
-	return resourceapply.ApplyDaemonSet(s.clientset.AppsV1(), events.NewInMemoryRecorder(componentName), ds, expectedGeneration, forceRollout)
+	return resourceapply.ApplyDaemonSetWithForce(s.clientset.AppsV1(), events.NewInMemoryRecorder(componentName), ds, expectedGeneration, forceRollout)
```

Comment:
Nit: I see half of the `sdkAPIUpdater` methods are not used anywhere, such as `applyDaemonSet`, `getDaemonSet`, `applyClusterRoleBinding`, `applyClusterRole`, `applyRoleBinding`, `applyRole` and maybe more. Can you please remove them in a subsequent PR?

Reply:
removed